src/api.tsx

```diff
@@ -19,6 +36,11 @@ const queryAPI = (
     headers: hasValidJWT()
       ? new Headers({ Authentication: `Bearer ${getJWT()}` })
       : undefined
+  }).catch((err: Error) => {
+    if (method !== 'GET') {
+      return addRequestToIdb({ path, method, body })
+    }
+    throw err
```

Comment:
Again, `else`.

Reply:
No

Reply:
For some reason github put this before my other comment about `else`